scripts/istio-utils.sh

```diff
@@ -70,16 +70,21 @@ function install_istio() {
     kubectl create ns istio-system || true
     helm template ${ISTIO_SRC}/install/kubernetes/helm/istio-init --name istio-init --namespace istio-system | kubectl apply -f -
 
+    # we need to wait for the custom resources installed by the previous command to be committed to the k8-api
+    crd_target_count=$(find ${ISTIO_SRC}/install/kubernetes/helm/istio-init/files -name "crd-[0-9]*" | xargs grep "kind: CustomResourceDefinition" | wc -l)
+
     ready=false
     for i in {1..12};do
         crd_count=$(kubectl get crds -n istio-system | grep 'istio.io\|certmanager.k8s.io' | wc -l)
-        if [[ ${crd_count} -gt 52 ]] ;then ready=true && break; fi
+        if [[ ${crd_count} -eq ${crd_target_count} ]] ;then ready=true && break; fi
```

Comment:
I would want to be conservative and not rule out the possibility the crd count might be greater for whatever reason, assuming given the case that the process is successful - that is, use `-gt` just in case.

Reply:
good catch, that could likely happen